client/src/components/Dashboard.tsx

```diff
@@ -18,10 +19,13 @@ export default function Dashboard () {
     'user',
     () => getUser(currentUser)
   );
+
+  // below need to give events a type, not any - need interface for event
   const { data: events, status: eventStatus } = useQuery(
```

Comment:
I learned you can destructure into an interface, e.g.

```suggestion
  const { data, status }: IEvent = useQuery(
```

This way you only define the types of `data`, `status`, etc. in the interface.

Reply:
will create a query interface